services/galley/test/integration/API/Util.hs

```diff
@@ -2923,6 +2935,38 @@ wsAssertAddProposal fromUser convId n = do
     getMLSMessageData (EdMLSMessage bs) = bs
     getMLSMessageData d = error ("Excepected EdMLSMessage, but got " <> show d)
 
+wsAssertCommit ::
```

Comment:
This went unused, right? Should it be added to that test that you have in this PR? If it stays unused, how about removing the utility altogether?

Reply:
nice catch!